doc/source/ray-core/walkthrough.rst

```diff
@@ -2,125 +2,144 @@
 
 .. _core-walkthrough:
 
-Ray Core Walkthrough
+Getting Started
 ====================
 
-This walkthrough will overview the core concepts of Ray:
+This tutorial will show you how to estimate the value of π using a `Monte Carlo method <https://en.wikipedia.org/wiki/Monte_Carlo_method>`_
+that works by randomly sampling points within a 2x2 square.
+We can use the proportion of the points that are contained within the unit circle centered at the origin
```

Comment:
May be include a picture here about the circle and all the computed values within that square as desirable estimation toward pi.
https://github.com/anyscale/ray-summit-2022-training/blob/main/ray-core/extra/highly_parallel.ipynb

Reply:
I feel the picture in https://medium.com/distributed-computing-with-ray/how-to-scale-python-multiprocessing-to-a-cluster-with-one-line-of-code-d19f242f60ff is more clear.

Reply:
excellent!